airbyte-config/init/src/main/resources/seed/source_definitions.yaml

```diff
@@ -1349,6 +1349,7 @@
   allowedHosts:
     hosts:
       - "${host}"
+      - "${tunnel_method.tunnel_host}"
```

Comment:
cc @bleonard / @prateekmukhedkar - this should be the complete set of needs for source-postgres!

Reply:
@evantahler I am working on adding `allowedHosts` to the remaining database sources([link](https://github.com/airbytehq/airbyte/issues/22372)). Reading through this PR, it appears that `${tunnel_method.tunnel_host}` is only required for those sources that support connection via SSH tunnels. (for example as of now we don't support connecting via ssh for source-redshift). And for all such sources I will only include `${host}` sub-value under `allowedHosts`. Does that sound correct?

Reply:
Yep!